src/main/java/stirling/software/SPDF/config/AppUpdateService.java

```diff
@@ -2,12 +2,14 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Scope;
 import org.springframework.stereotype.Service;
 
 import stirling.software.SPDF.config.interfaces.ShowAdminInterface;
 import stirling.software.SPDF.model.ApplicationProperties;
 
+@Configuration
 @Service
```

Comment:
If we are adding configuration tag we should remove service

Reply:
Yes you are right